example-get-started/generate.sh

```diff
@@ -76,6 +77,14 @@ init_remote_storage() {
     dvc remote add -f -d $OPT_REMOTE ssh://dvc@35.194.53.251/home/dvc/storage
     dvc remote modify $OPT_REMOTE keyfile /Users/ivan/.ssh/dvc_gcp_remotes_rsa
   fi
+
+  if [ $OPT_REMOTE == 'private-azure' ]; then
+    # Make sure that you have connection string in your env or some other way
+    # provide credentials for the `dvcprivate` storage account. Copy the connection
+    # string from the Azure portal and export it with
+    # `AZURE_STORAGE_CONNECTION_STRING`
+    dvc remote add -f -d $OPT_REMOTE azure://nlp
```

Comment:
minor: should we use a more descriptive name for the bucket?

Reply:
tbh, was thinking to migrate our example repos to something like `nlp`, `cv`, etc. But we can come up with some better names of course. For Azure - let's keep it as-is since I don't think it will become an official example or anything.